cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -141,11 +142,20 @@ jobs:
           echo "paths<<EOF" >> "$GITHUB_OUTPUT"
           jq --raw-output ".artifacts[]?.path | select( . != null )" dist-manifest.json >> "$GITHUB_OUTPUT"
           echo "EOF" >> "$GITHUB_OUTPUT"
+
+          cp dist-manifest.json "$BUILD_MANIFEST_NAME"
       - name: "Upload artifacts"
         uses: actions/upload-artifact@v3
         with:
           name: artifacts
-          path: ${{ steps.cargo-dist.outputs.paths }}
+          path: |
+            ${{ steps.cargo-dist.outputs.paths }}
+      - name: "Upload manifest"
+        uses: actions/upload-artifact@v3
+        with:
+          name: artifacts
+          path: |
+            ${{ env.BUILD_MANIFEST_NAME }}
```

Comment:
if it works, copying the manifest to target/distrib/ and just doing one upload seems simpler/cleaner

Reply:
I don't want to deal with canonicalizing the path so both branches of the `path` are absolute. The environment variable's assigned outside bash

Reply:
i don't think you need to canonicalize the path?

Reply:
I should have just trusted it'd work, looks like it is happy with that!